src/proto/beats.proto

```diff
@@ -23,7 +23,7 @@ message TimeSignature {
 }
 
 message Beat {
-  optional int32 frame_position = 1;
+  optional double frame_position = 1;
```

Comment:
I assumed frames should be integers. When reading from an audio file, a frame position is integral.
Does the double-precision account for the adjustments made by mixxx engine?

Reply:
The transport engine in Mixxx has double precision. This is required to set a beat in a 48kHz Track that is played with 44.1kHz. 

It is also required to remove the quantization error (jitter) of a beat, because it may snap to an earlier or later sample. 

In this special case moving to double precision fixed some test using 130 BPM
where beats are at fractional frames.